### ISSUE TITLE
1.16.2 mods are ignored when updating/refreshing

### ISSUE THREAD

**Issue by martmists** [calloatti/modrobot#15]
And yes, I have set maxversion to 1.16.2

**Comment by calloatti** [comment]
The problem is the CurseForge API that does not send 1.16.2 data until there is a proper Forge Mod loader release for 1.16.2, otherwise their own Twitch launcher breaks.

Check for example this file: https://addons-ecs.forgesvc.net/api/v2/addon/396505/file/3029695

```
"gameVersion": ["Fabric"],
```

And on the CF site: https://www.curseforge.com/minecraft/mc-mods/autoswap-updated/files/3029695

As you can see, the file is tagged with '1.16.2'  on the site, but the API does not send the data. 

TLDR: CF is broken and this happens every time there is a new MC release.
